### ISSUE TITLE
Visualizer error codes on OSX

### ISSUE THREAD

**Issue by gabrielcarino** [scottlawsonbc/audio-reactive-led-strip#195]
Hi I'm pretty new to coding and can't figure out what went wrong, does on anyone have suggestions?
![screen shot 2019-02-17 at 7 00 04 pm](https://user-images.githubusercontent.com/47729949/52926067-1b66c700-32e9-11e9-9115-feb4913e4e36.png)
![screen shot 2019-02-17 at 6 59 45 pm](https://user-images.githubusercontent.com/47729949/52926068-1b66c700-32e9-11e9-89b4-93a21005aa09.png)

**Comment by joeybab3** [comment]
Did you install the dependencies as shown in the instructions?

**Comment by gabrielcarino** [comment]
> Did you install the dependencies as shown in the instructions?

"Visualization code is compatible with Python 2.7 or 3.5. A few Python dependencies must also be installed:

Numpy
Scipy (for digital signal processing)
PyQtGraph (for GUI visualization)
PyAudio (for recording audio with microphone)"
Is this what I need?

**Comment by joeybab3** [comment]
Yes, please follow the instructions and install all the dependencies before trying to run it.

**Comment by gabrielcarino** [comment]
I was able to install the first 3 dependencies however the final one is giving me error codes, I think something to do with a virtual environment?
![screen shot 2019-02-17 at 6 59 45 pm](https://user-images.githubusercontent.com/47729949/52928174-46eeaf00-32f3-11e9-84ef-ce93927b9933.png)


> > Did you install the dependencies as shown in the instructions?
> 
> "Visualization code is compatible with Python 2.7 or 3.5. A few Python dependencies must also be installed:
> 
> Numpy
> Scipy (for digital signal processing)
> PyQtGraph (for GUI visualization)
> PyAudio (for recording audio with microphone)"
> Is this what I need?

**Comment by gabrielcarino** [comment]
Hi again, I have no what I'm doing wrong
![screen shot 2019-02-18 at 10 07 57 am](https://user-images.githubusercontent.com/47729949/52969549-9c63a400-3365-11e9-9caa-79b31df2c966.png)
![screen shot 2019-02-18 at 10 09 01 am](https://user-images.githubusercontent.com/47729949/52969551-9d94d100-3365-11e9-8a2e-1c11410c83c8.png)
![screen shot 2019-02-18 at 10 10 28 am](https://user-images.githubusercontent.com/47729949/52969553-9f5e9480-3365-11e9-80c0-ee249bde7e74.png)
![screen shot 2019-02-18 at 10 11 26 am](https://user-images.githubusercontent.com/47729949/52969558-a1c0ee80-3365-11e9-98e2-5542df45b808.png)

**Comment by joeybab3** [comment]
Sounds like PyAudio didn't install correctly then.

**Comment by gabrielcarino** [comment]
I've been using pip to install the dependencies, is there another method that might work better?

**Comment by joeybab3** [comment]
What are you doing this on, I'm assuming a mac?

**Comment by gabrielcarino** [comment]
yes

**Comment by joeybab3** [comment]
I don't have a mac to test it but i can try and figure it out later

**Comment by scottlawsonbc** [comment]
How are you running the python code? It's strange that it can't find the other files in the repository.

**Comment by gabrielcarino** [comment]
I've just been running them from my desktop, but I even tried running in out of the same directory as the dependencies...
